app/components/Reactions/Reaction.tsx

```diff
@@ -60,7 +62,7 @@ const Reaction = ({
   }
 
   let tooltipContent = '';
-  if (users && users.length > 0) {
+  if (showPeople && users && users.length > 0) {
```

Comment:
Isn't `users` `null` on articles and quotes etc.? Why isn't this enough? `showPeople` seems overkill

Reply:
The problem is that redux makes `users` not null (it containts your reaction) when you react on an article/quote I think. So that will make the tooltip show only your reaction when you react in those cases.

Reply:
I am quite new to Redux and React in general, so let me know if you have a better solution for this @ivarnakken 😊

Reply:
ah yes you're right, it makes sense that Redux will store your own reaction 👍🏼